### PR TITLE
Change visitant status to not be removed upon KO

### LINKED ISSUE #1733: In abyssea player death is teleported to the entrance incorrectly
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: EDITME

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

When the player is active in Abysssea and dies, the player's corpse will be forcibly teleported to the entrance outside Abysssea

### PR DESCRIPTION
**Pull request by illzbee**
**_I affirm:_**
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I've _**tested my code and the things my code has changed**_ since the last commit in the PR, and will test after any later commits

## What does this pull request do?

Change visitant status to not be removed upon KO
Closes #1733

### CODE REVIEW

File: sql/status_effects.sql [hunk 1]
```diff
@@ -314,7 +314,7 @@ INSERT INTO `status_effects` VALUES (281,'enthunder_ii',41,94,51,0,0,0,5,0,0);
 INSERT INTO `status_effects` VALUES (282,'enwater_ii',41,94,51,0,0,0,6,0,0);
 INSERT INTO `status_effects` VALUES (283,'perfect_defense',32,0,0,0,0,0,0,0,0);
 INSERT INTO `status_effects` VALUES (284,'egg',32,0,0,0,0,0,0,0,0);
-INSERT INTO `status_effects` VALUES (285,'visitant',32,0,0,0,0,0,0,0,0);
+INSERT INTO `status_effects` VALUES (285,'visitant',33554432,0,0,0,0,0,0,0,0);
```
Comment (TeoTwawki) [inline sql/status_effects.sql]: This should be 41943296 to include loss on being zoned out and prevent being clicked off via add-ons.

**Review by TeoTwawki — CHANGES_REQUESTED** [review]

File: sql/status_effects.sql (hunk 1 above)
Comment (illzbee) [reply sql/status_effects.sql]: Ok I will try that out, this is non-removable by the user, lost upon zoning out, and is removed via time out I will check with that number.

Tested works great! Thanks!

Comment (TeoTwawki) [reply sql/status_effects.sql]: src\map\status_effects.h has the flags enumerated in hex. Can add them up and convert to decimal in windows windows calculator (or similar) to see the combined value for here.

Comment (illzbee) [reply sql/status_effects.sql]: I had no idea "being clicked off via add-ons" was a thing

**Review by claywar — APPROVED** [review]

**Review by zach2good — APPROVED** [review]

**Review by TeoTwawki — APPROVED** [review]

**Pull request merged**